config/quality/pmd/pmd-ruleset.xml

```diff
@@ -20,8 +20,8 @@
     <rule ref="rulesets/java/naming.xml">
         <!--<exclude name="AbstractNaming" />-->
         <exclude name="LongVariable" />
-        <!--<exclude name="ShortMethodName" />-->
-        <!--<exclude name="ShortVariable" />-->
+        <exclude name="ShortMethodName" />
```

Comment:
I got it. I will revert the modification 😆 for pmd rule.

Maybe some Realm model classes will have `@SuppressWarning("PMD.ShortVariable")` because DDP often uses short JSON keys like `id` or `t` and they are mapped to Realm object as it is.